modules/programs/fzf.nix

```diff
@@ -141,7 +163,8 @@ in {
         FZF_CTRL_T_COMMAND = cfg.fileWidgetCommand;
         FZF_CTRL_T_OPTS = cfg.fileWidgetOptions;
         FZF_DEFAULT_COMMAND = cfg.defaultCommand;
-        FZF_DEFAULT_OPTS = cfg.defaultOptions;
+        FZF_DEFAULT_OPTS = cfg.defaultOptions
+          ++ [ "--color ${renderedColors cfg.colors}" ];
```

Comment:
This doesn't work when there are no colors specified.

Reply:
Oh yes indeed, fixed.